package.json

```diff
@@ -12,7 +12,7 @@
     "dev": "tsc --watch",
     "build": "tsc",
     "dev-example": "webpack-dev-server --config example/webpack.config.js --inline --hot",
-    "build-example": "rm -rf example/dist && webpack --config example/webpack.config.js --env.prod",
```

Comment:
We could simply use [rimraf](https://www.npmjs.com/package/rimraf) without adding a webpack plugin

Reply:
If so the build-example script should be 
`rimraf -rf example/dist && webpack --config example/webpack.config.js --env.prod`